debian/changelog

```diff
@@ -14,13 +14,13 @@ cloud-init (23.4.1-0ubuntu1~20.04.3) UNRELEASED; urgency=medium
     [1] commit/b519d861aff8b44a0610c176cb34adcbe28df144
   * refresh patches:
     - d/p/netplan99-cannot-use-default.patch
+    - d/p/retain-ec2-default-net-update-events.patch
     - d/p/retain-netplan-world-readable.patch
     - d/p/status-do-not-remove-duplicated-data.patch
     - d/p/status-retain-recoverable-error-exit-code.patch
-  * Upstream snapshot based on upstream/main at c948e418.
-  * Upstream snapshot based on upstream/main at a4140119.
+  * Upstream snapshot based on 6d61b63 at 6d61b63f.
```

Comment:
Was this intentional? Why not `on main`? Either way, shouldn't hurt anything, so +1.

Reply:
> Was this intentional? Why not on main? Either way, shouldn't hurt anything, so +1.

Yes, because I wanted this release to match jammy and mantic. They should all have released on the same commit.